File: Fibonacci1.py
def main():
    for i in range(20):
        print(generate(i) ,  end = " ")   
        i += 1
    
#Recursive function to generate fibonacci sequence.
def generate(num): 
    if num == 0:    
        return num
    
    elif num == 1:    
       return 1

    else:    
        #Recursion
        num1 = generate(num - 1) + generate(num - 2)
        return num1
    
#call the main function.
main()

File: Fibonacci1.c
#include <stdio.h>

int generate(int num);
int main() 
{
    for (int i = 0; i < 20; i++) 
    {
        printf("%i", generate(i));
        printf(" ");     
    }
}

//Recursive function to generate fibonacci sequence.
int generate(int num) 
{
    if (num == 0) 
    {
        return num;
    } 
	else if (num == 1) 
    {
        return 1;
    } 
	else 
    {
        //Recursion
        return (generate(num - 1) + generate(num - 2));  
    }
}
